# HG changeset patch
# User Rik <rik@octave.org>
# Date 1517025004 28800
#      Fri Jan 26 19:50:04 2018 -0800
# Node ID e04a56630c8a7cfdfc081b951625fbfccca5d3fa
# Parent  78aff6f142271e21e839e298fe88500f0c11fd13
variable-editor-model.cc: Fix missing parenthesis preventing compilation.

* variable-editor-model.cc (type_is_editable): Fix missing parenthesis
preventing compilation.

diff --git a/libgui/src/variable-editor-model.cc b/libgui/src/variable-editor-model.cc
--- a/libgui/src/variable-editor-model.cc
+++ b/libgui/src/variable-editor-model.cc
@@ -752,17 +752,17 @@ variable_editor_model::clear_data_cell (
 {
   m_d->clear (r, c);
 }
 
 bool
 variable_editor_model::type_is_editable (const octave_value& val,
                                          bool display_error) const
 {
-  if ((val.isnumeric () || val.islogical () || val.iscell ()
+  if ((val.isnumeric () || val.islogical () || val.iscell ())
       && val.ndims () == 2)
     return true;
 
   if (display_error)
     {
       QString tname = QString::fromStdString (val.type_name ());
 
       dim_vector dv = val.dims ();
